camp/jbolotski/beam_break_motor/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.common.Types.BeamBreakState;
+import frc.robot.sim.PhysicsSim;
+import frc.robot.subsystems.BeamBreakMotor;
+import frc.robot.subsystems.BeamBreakSensor;
+
+/**
+ * This class is where the bulk of the robot should be declared. Since Command-based is a
+ * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
+ * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
+ * subsystems, commands, and button mappings) should be declared here.
+ */
+public class RobotContainer {
+  // The robot's subsystems are defined here...
+  private final BeamBreakMotor m_beamBreakMotor = new BeamBreakMotor(Constants.beamBreakMotorSpeed);
+  private BeamBreakSensor m_beamBreakSensor = new BeamBreakSensor(Constants.beamBreakSensorPort);
+
+  // The robot's commands are defined here...
+
+  /** The container for the robot. Contains subsystems, OI devices, and commands. */
+  public RobotContainer() {
+    // Configure the button bindings
+    configureButtonBindings();
+  }
+
+  /**
+   * Use this method to define your button->command mappings. Buttons can be created by
+   * instantiating a {@link GenericHID} or one of its subclasses ({@link
+   * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
+   * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
+   */
+  private void configureButtonBindings() {
+    m_beamBreakSensor.whenUnbroken(() -> m_beamBreakMotor.runMotor(), m_beamBreakMotor);
```

Comment:
So I created a subbranch of yours.  See my PR at: https://github.com/Team4682CyBears/ChargedUp2023/pull/14

The idea there is generally something like (especially true for teleop but also for auto in this case):

1. Have input change events trigger creation of a new command
2. Commands then direct subsystems to do a succession of operations (in this case its only one)
3. Subsystems then translate a high level operation into actions at the hardware level (motors, solenoids, lights, music)

Take a look and let me know what you think.